.azure-pipelines/testbed_health_check.py

```diff
@@ -303,22 +310,40 @@ def check_bgp_session_state(self, state="established"):
         logger.info("======================= check_bgp_session_state starts =======================")
 
         for sonichost in self.sonichosts:
+            if (self.is_multi_asic and
+                    self.duts_basic_facts[sonichost.hostname]["ansible_facts"]["dut_basic_facts"]["is_supervisor"]):
+                logger.info("Skip check_bgp_session_state on Supervisor.")
+                continue
```

Comment:
BGP check should be skipped on Supervisor if T2 as there's no BGP on Supervisor.